extensions/container-image/container-image-docker/deployment/src/test/resources/openjdk-21-runtime

```diff
@@ -0,0 +1,18 @@
+# Use Java 17 base image
```

Comment:
Should this comment not be Java 21?

Reply:
Yes, fancy a PR?

Reply:
Maybe let's just drop these comments? They have no value really and are error prone.

Reply:
> Maybe let's just drop these comments? They have no value really and are error prone.

+1